bcda/api/requests_test.go

```diff
@@ -123,159 +127,80 @@ func (s *RequestsTestSuite) TestRunoutDisabled() {
 	s.Contains(string(body), "Invalid group ID")
 }
 
-func (s *RequestsTestSuite) TestInvalidRequests() {
```

Comment:
Replaced by test in [validation_test.go](https://github.com/CMSgov/bcda-app/pull/673/files#diff-a03a477e3e4270213962d11eb6f2f8c5386f25cac2ac4976a001dbf8d0fbd45cR43)